llvm/lib/Target/AMDGPU/MCTargetDesc/CMakeLists.txt

```diff
@@ -20,6 +21,7 @@ add_llvm_component_library(LLVMAMDGPUDesc
   CodeGenTypes
   Core
   MC
+  MCParser
```

Comment:
Questioning whether it's desired to have `MCParser` within the `AMDGPUDesc`. Initially seemed suitable to have `AMDGPUMCKernelCodeT` together with `AMDGPUMCKernelDescriptor` as they are both consumers of `SIProgramInfo` and serve a similar metadata emit purpose but it may be desired to have `AMDGPUMCKernelCodeT` elsewhere. Let me know if anybody objects to the `MCParser` and/or would prefer `AMDGPUMCKernelCodeT` to live elsewhere.

Reply:
Moved the `MCTargetDesc/AMDGPUMCKernelCodeT.*` files to the original `Utils/AMDKernelCodeTUtils.*` files and location. Should now adhere to the same access to parsing as prior this change.